sdks/nodejs/src/agonesSDK.js

```diff
@@ -14,15 +14,20 @@
 
 const grpc = require('@grpc/grpc-js');
 
+const Alpha = require('./alpha');
+
 const messages = require('../lib/sdk_pb');
 const servicesPackageDefinition = require('../lib/sdk_grpc_pb');
 
 class AgonesSDK {
 	constructor() {
 		const services = grpc.loadPackageDefinition(servicesPackageDefinition);
-		this.client = new services.agones.dev.sdk.SDK(`localhost:${this.port}`, grpc.credentials.createInsecure());
+		const address = `localhost:${this.port}`;
+		const credentials = grpc.credentials.createInsecure();
+		this.client = new services.agones.dev.sdk.SDK(address, credentials);
 		this.healthStream = undefined;
 		this.streams = [];
+		this.alpha = new Alpha(address, credentials);
```

Comment:
Curious - any reason for `sdk.alpha` as a property, as opposed to `sdk.alpha()`

Just wondering if `sdk.alpha` could be overwritten by the user?

Reply:
`sdk.alpha`  could be overwritten even if it is a method/function, but only on the instance

We could add an alpha method and would need to rename the property in JS but then might as well convert to a getter but at that point we have a getter for a public property which is redundant, and the getter could still be overwritten (as well as the property). There are ways of making things private but nothing straightforward so I would go with public which seems to be the JS way of doing things.

Reply:
Cool! Thanks for the explanation. That makes perfect sense :+1: